src/Screenshot.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
```

Comment:
in generale prendi in considerazione di riorganizzarlo come classe, o in alternativa potresti considerare di trasformare _load_image_ in un decoratore per _take_screenshot_ cosi' da rendere tutto piu' pythonoso